### PR TITLE
Update the `kolide_spotlight` table with a longer timeout

### PR DESCRIPTION
**Pull request by directionless**
Relates to https://github.com/kolide/k2/issues/9493

To address an `mdfind` issue, this PR does a couple of things:
1. It ups the mdfind exec timeout. I'm a little wary here, but we see cases where 60s time out
2. It moves the `kolide_spotlight` table to using the tablehelpers.Exec. 
3. TODO: improve the exec helper's timing info

### CODE REVIEW

File: pkg/osquery/interactive/interactive_test.go [hunk 1]
```diff
@@ -14,6 +14,7 @@ import (
 	"runtime"
 	"testing"
 
+	"github.com/go-kit/log"
```
Comment (RebeccaMahany) [inline pkg/osquery/interactive/interactive_test.go:17]:
```suggestion
	"github.com/go-kit/kit/log"
```

Comment (RebeccaMahany) [reply pkg/osquery/interactive/interactive_test.go:17]: (will fix the build error)

**Review by James-Pickett — DISMISSED** [review]
LGTM!

File: pkg/osquery/interactive/interactive_test.go (hunk 1 above)
Comment (directionless) [reply pkg/osquery/interactive/interactive_test.go]: :shakes fist at vcode:

**Review by RebeccaMahany — APPROVED** [review]

**Pull request merged**